gaia/tasks.py

```diff
@@ -0,0 +1,42 @@
+from celery import Celery
+from gaia.parser import parse_request
+from gaia.core import config
+
+app = Celery('tasks',
+             backend=config['gaia']['celery_backend'],
+             broker=config['gaia']['celery_broker'])
+
+
+@app.task
+def parse_process(process_name, request_json):
+    """
```

Comment:
@kotfic, the purpose of the GaiaRequestParser is to take user input via a POST request, generate a GaiaProcess object and it's input objects from the request parameter and JSON body, run the process, and return the output.   The parse_process task is intended to have all that take place asynchronously on a celery worker, just passing the POST args/request body on to celery.  But if it makes more sense then I could have a celery task that given an existing GaiaProcess object with args and inputs already defined, do just the computation via celery and return the output.  [execute_process] (https://github.com/OpenDataAnalytics/gaia/pull/44/files#diff-7e93e316c496a03dc3158177c189a635R25) is halfway there, I could get rid of the inputs and args parameters, assuming they have already been defined for the process.  But that Process would need to be created first somehow, either programmatically or via the GaiaRequestParser.  I could add a boolean argument to the parser to determine whether to have the process run its compute method via celery task or not.